docs/docs/deployment/hosting/kubernetes.md

```diff
@@ -277,184 +292,184 @@ values to set to configure the task processor using the helm chart.
+| `taskProcessor.readinessProbe.successThreshold`    |                                                                           | 1                              |
+| `taskProcessor.readinessProbe.timeoutSeconds`      |                                                                           | 2                              |
+| `taskProcessor.podAnnotations`                     |                                                                           | `{}`                           |
+| `taskProcessor.resources`                          |                                                                           | `{}`                           |
+| `taskProcessor.podLabels`                          |                                                                           | `{}`                           |
+| `taskProcessor.nodeSelector`                       |                                                                           | `{}`                           |
+| `taskProcessor.tolerations`                        |                                                                           | `[]`                           |
+| `taskProcessor.affinity`                           |                                                                           | `{}`                           |
+| `taskProcessor.podSecurityContext`                 |                                                                           | `{}`                           |
+| `taskProcessor.defaultPodSecurityContext.enabled`  | whether to use the default security context                               | `true`                         |
+| `postgresql.enabled`                               | if `true`, creates in-cluster PostgreSQL database                         | `true`                         |
+| `postgresql.serviceAccount.enabled`                | creates a serviceaccount for the postgres pod                             | `true`                         |
+| `nameOverride`                                     |                                                                           | `flagsmith-postgres`           |
+| `postgresqlDatabase`                               |                                                                           | `flagsmith`                    |
+| `postgresqlUsername`                               |                                                                           | `postgres`                     |
+| `postgresqlPassword`                               |                                                                           | `flagsmith`                    |
+| `databaseExternal.enabled`                         | use an external database. Specify database URL, or all parts.             | `false`                        |
+| `databaseExternal.url`                             | See [schema](https://github.com/kennethreitz/dj-database-url#url-schema). |                                |
```

Comment:
This was an existing issue that I fixed as part of this PR. Unfortunately, it means this PR is larger than it needs to be because it changed the formatting of the rest of the rest of the table, but I think it's worth keeping in.